readme.md

```diff
@@ -72,6 +98,59 @@ Temporal is a [durable execution system](https://temporal.io/blog/building-relia
 
 - [iWF](https://github.com/indeedeng/iwf) - DSL workflow framework.
 
+## Videos
+
+### Why Temporal
+
+- [Replay Keynote](https://www.youtube.com/watch?v=W0Ygep6iCJY) - Durable execution, Nexus, and Cloud.
+- [Glovo: Fault tolerant distributed microservices with Temporal](https://youtu.be/6lSuDRRFgyY)
+- [TheDevConf: Fault Tolerant, Distributed Microservices Orchestration with Temporal](https://www.youtube.com/watch?v=6T6zVZHU7_Q) - Introduction to Temporal ([slides](https://temporal-intro-and-demo.netlify.app/)).
+- [State of Affairs or Affairs of State](https://www.youtube.com/watch?v=2P_aXee2qh4) - Stateless vs stateful architecture and why Temporal.
```

Comment:
depends on how focused you want these resources to be but if only part of this answers the "why temporal" maybe just link to the timestamp where the presenter starts talking about that.